File: platforms/web/examples/scene/material/alpha_mode.js
import { AlphaMode, Material } from "fragmentcolor";

const foliage = Material.pbr().alphaMode(AlphaMode.Mask).alphaCutoff(0.3);

const glass = Material.pbr().baseColor([0.9, 0.95, 1.0, 0.25]).alphaMode(AlphaMode.Blend);

const solid = Material.pbr().alphaMode(AlphaMode.Opaque);

File: platforms/swift/examples/scene/material/alpha_mode.swift
import FragmentColor

let foliage = Material.pbr().alphaMode(AlphaMode.mask).alphaCutoff(0.3)

let glass = try Material.pbr().baseColor([0.9, 0.95, 1.0, 0.25]).alphaMode(AlphaMode.blend)

let solid = Material.pbr().alphaMode(AlphaMode.opaque)